group-coordinator/src/test/java/org/apache/kafka/coordinator/group/consumer/TargetAssignmentBuilderTest.java

```diff
@@ -128,10 +129,10 @@ public void updateMemberSubscription(
             Optional<String> instanceId,
             Optional<String> rackId
         ) {
-            ConsumerGroupMember existingMember = members.get(memberId);
+            GroupMember existingMember = members.get(memberId);
             ConsumerGroupMember.Builder builder;
             if (existingMember != null) {
-                builder = new ConsumerGroupMember.Builder(existingMember);
+                builder = new ConsumerGroupMember.Builder((ConsumerGroupMember) existingMember);
```

Comment:
This is exactly what I don't want to see.

Reply:
Agree, resolved.